cs/v1/CS_v1.0.0.yaml

```diff
@@ -645,15 +725,11 @@ components:
               UNLO: '#/components/schemas/UNLocationLocation'
         vesselSchedules:
           type: array
-          x-stoplight:
-            id: oqtnha66e947j
           minItems: 1
```

Comment:
Is `vesselSchedules` a required field? If not - then please remove the `minItems: 1`